contracts/smart-account/modules/PasskeyRegistryModule.sol

```diff
@@ -25,20 +25,20 @@ contract PasskeyRegistryModule is
     string public constant NAME = "PassKeys Ownership Registry Module";
     string public constant VERSION = "0.2.0";
 
-    mapping(address => PassKeyId) public smartAccountPassKeys;
+    mapping(address => PassKeyId) public _smartAccountPasskey;
```

Comment:
`_` according to solidity best practices and guidelines should be used only for `private` and `internal` variables / functions